source/_reference-g-s.html.md.erb

```diff
@@ -8346,6 +8348,17 @@ Web Content | **WEBCONTENT** | [WebContentWidgetContent](#webcontentwidgetconten
 **hyperlink** | [WidgetHyperlink](#widgethyperlink-object) | The widget has **when clicked** attribute set to that hyperlink (if present and non-null). Hyperlinks will have **interactionType**.
 **width** | integer | Original width of the image in pixels
 
+### ProfileField Object
+
+     |     |     |
+-----|-----|-----|
+**id** | number | Column Id
+**index** | number | Column index or position. This number is zero-based. 
+**title** | string | Column title
+**type** | string | One of:<ul><li>**ABSTRACT_DATETIME**</li><li>**CHECKBOX**</li><li>**CONTACT_LIST**</li><li>**DATE**</li><li>**DATETIME**</li><li>**DURATION**</li><li>**MULTI_CONTACT_LIST**</li><li>**MULTI_PICKLIST**</li><li>**PICKLIST**</li><li>**PREDECESSOR**</li><li>**TEXT_NUMBER**</li></ul>See [Column Types](#column-types).
+**locked** | Boolean | Indicates whether the column is locked. In a response, a value of **true** indicates that the column has been locked by the sheet owner or the admin.
+**lockedForUser** | Boolean | Indicates whether the column is locked for the requesting user. This attribute may be present in a response, but cannot be specified in a request.
+
```

Comment:
We'll want to include the following, as we do for the "cell" type:

- objectValue
- displayValue
- format
- formula
- hyperlink
- image

We'll also want to include:

- options (Array of the options available for the field)
- symbol (When applicable for CHECKBOX or PICKLIST column types. See Symbol Columns.)

Reply:
Ah, documented already as SummaryField object.